APMrover2/mode_guided.cpp

```diff
@@ -145,6 +145,33 @@ bool ModeGuided::reached_destination() const
     return true;
 }
 
+// get desired location
+bool ModeGuided::get_desired_location(Location& destination)
+{
+    switch (_guided_mode) {
+    case Guided_WP:
+        if (g2.wp_nav.is_destination_valid()) {
+            destination = g2.wp_nav.get_destination();
+            return true;
+        } else {
```

Comment:
remove else and break, just return false

Reply:
removed, txs